patches/ungoogled-chromium/windows/windows-fix-fingerprinting.patch

```diff
@@ -1,8 +1,39 @@
 # Fixes linking errors associated with the fingerprinting switches added by ungoogled-chromium.
 
+--- a/components/ungoogled/ungoogled_switches.cc
++++ b/components/ungoogled/ungoogled_switches.cc
+@@ -6,6 +6,7 @@
+ 
+ namespace switches {
+ 
++#if !defined(OS_WIN)
+ // Enable fingerprinting deception for getClientRects and getBoundingClientRect
+ const char kFingerprintingClientRectsNoise[] = "fingerprinting-client-rects-noise";
+ 
+@@ -14,5 +15,6 @@ const char kFingerprintingCanvasMeasureT
+ 
+ // Enable fingerprinting deception for Canvas image data
+ const char kFingerprintingCanvasImageDataNoise[] = "fingerprinting-canvas-image-data-noise";
++#endif
+ 
+ }  // namespace switches
+--- a/components/ungoogled/ungoogled_switches.h
++++ b/components/ungoogled/ungoogled_switches.h
+@@ -11,9 +11,11 @@
+ 
+ namespace switches {
+ 
++#if !defined(OS_WIN)
+ COMPONENT_EXPORT(UNGOOGLED) extern const char kFingerprintingClientRectsNoise[];
+ COMPONENT_EXPORT(UNGOOGLED) extern const char kFingerprintingCanvasMeasureTextNoise[];
+ COMPONENT_EXPORT(UNGOOGLED) extern const char kFingerprintingCanvasImageDataNoise[];
```

Comment:
Since I added `COMPONENT_EXPORT(UNGOOGLED)` to these declarations, would the build work without this patch?

Reply:
Testing out building the current version of Ungoogled Chromium for Windows without this patch. Will let you know if the build succeeds and if it fixes the issue.

Reply:
Without windows-fix-fingerprinting.patch, the build succeeds. However, it doesn't fix the install prompt issue for Ungoogled Chromium 80.0.3987.149

Reply:
Can anyone please explain to me how to update this browser, I do not know code, but I want my data protected,I've spent few hours here cand seem to find anything, I don't know ehere to put the archives, since there is now exe.
Please take your time an help me :D
The extensions also don't work, tried every method. it still says "adding to chromnium" and nothing happens